python/openmldb/driver.py

```diff
@@ -59,6 +59,13 @@ def dropDB(self, db):
             return True, "ok"
         else:
             return False, status.msg
+        
+    def getAllTables(self):
+        if not self.sdk:
+            return False, "please init driver first"
```

Comment:
Should not return different types in same method. You may simply return empty list here or throw an exception.

Reply:
updated.